_config.yml

```diff
@@ -949,19 +949,19 @@ vendors:
 
   # Mermaid
   # mermaid: //unpkg.com/mermaid@8/dist/mermaid.min.js
-  # mermaid: //cdnjs.cloudflare.com/ajax/libs/mermaid/8.5.2/mermaid.min.js
+  # mermaid: //cdnjs.cloudflare.com/ajax/libs/mermaid/8.6.3/mermaid.min.js
   mermaid:
 
-  # Internal version: 3.1.1
```

Comment:
There is no need to update the CDN version of `animate_css`. See the description in https://github.com/next-theme/hexo-theme-next/pull/41
Thanks!